forwarder/update_handlers/pva_update_handler.py

```diff
@@ -106,14 +114,19 @@ def _monitor_callback(self, response: Union[Value, Exception]):
                     self._repeating_timer.reset()
 
     def publish_cached_update(self):
-        with self._cache_lock:
-            if self._cached_update is not None:
-                # Always include current alarm status in periodic update messages
-                self._publish_message(
-                    *self._message_serialiser.serialise(
-                        self._cached_update, serialise_alarm=True
+        try:
+            with self._cache_lock:
+                if self._cached_update is not None:
+                    # Always include current alarm status in periodic update messages
+                    self._publish_message(
+                        *self._message_serialiser.serialise(
+                            self._cached_update, serialise_alarm=True
+                        )
                     )
-                )
+        except (RuntimeError, ValueError) as e:
```

Comment:
Just for curiosity `RunTimeError and ValueError` are expected to be raised during serialization?

Reply:
Those are the exceptions that I will know can be raised. I do not want to catch exceptions I don't know can be raised.